apertium-eng.eng.tsx

```diff
@@ -1,4 +1,18 @@
 <?xml version="1.0" encoding="UTF-8"?>
+<!--
```

Comment:
This breaks training the tagger. It seems it doesn't like top-level comments in embedded XMLs (see https://github.com/apertium/apertium-eng/blob/master/apertium-eng.eng.mtx#L21)